### PR TITLE
Docs: Add OAuth improvements to what's new in 9.3

### PR DESCRIPTION
**Pull request by mgyongyosi**
**What is this feature?**
This PR extends the `What's new in 9.3` page with the OAuth token handling improvements.

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v9-3.md [hunk 1]
```diff
@@ -73,3 +73,13 @@ With the Partition by values transformer, you can now issue a single query and s
 | ------------------- | ------ | ----- |
 | 2022-10-20 12:00:00 | EU     | 2936  |
 | 2022-10-20 01:00:00 | EU     | 912   |
+
+## Authentication - OAuth token handling improvements
+
+Generally available in Grafana Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced.
```
Comment (chri2547) [inline docs/sources/whatsnew/whats-new-in-v9-3.md:79]:
```suggestion
This feature is generally available in Grafana Open Source, Enterprise, Cloud Free, Cloud Pro, and Cloud Advanced.
```

**Review by Jguer — APPROVED** [review]

File: docs/sources/whatsnew/whats-new-in-v9-3.md [hunk 2]
```diff
@@ -73,3 +73,13 @@ With the Partition by values transformer, you can now issue a single query and s
 | ------------------- | ------ | ----- |
 | 2022-10-20 12:00:00 | EU     | 2936  |
 | 2022-10-20 01:00:00 | EU     | 912   |
+
+## Authentication - OAuth token handling improvements
+
+This feature is generally available in Grafana Open Source, Enterprise, Cloud Free, Cloud Pro, and Cloud Advanced.
+
+As part of our efforts to improve security of Grafana, we introduce a long-awaited feature which enhances Grafana's OAuth 2.0 compatibility. When a user logs in using an OAuth provider, Grafana on each request verifies that the access token has not expired. When an access token expires, Grafana uses the provided refresh token (if any exists) to obtain a new access token.
```
Comment (mjseaman) [inline docs/sources/whatsnew/whats-new-in-v9-3.md:81]:
```suggestion
As part of our efforts to improve security of Grafana, we introduce a long-awaited feature which enhances Grafana's OAuth 2.0 compatibility. When a user logs in using an OAuth provider, Grafana verifies on each request that the access token has not expired. When an access token expires, Grafana uses the provided refresh token (if any exists) to obtain a new access token.
```

File: docs/sources/whatsnew/whats-new-in-v9-3.md [hunk 3]
```diff
@@ -73,3 +73,13 @@ With the Partition by values transformer, you can now issue a single query and s
...
+
+This feature is behind the `accessTokenExpirationCheck` feature toggle and it is disabled by default.
+
+Complete documentation on how to configure obtaining a refresh token can be found on the specific Identity Provider's [configuration page]({{< relref "../setup-grafana/configure-security/configure-authentication/" >}}).
```
Comment (mjseaman) [inline docs/sources/whatsnew/whats-new-in-v9-3.md:85]:
```suggestion
Complete documentation on how to configure obtaining a refresh token can be found on the [authentication configuration page]({{< relref "../setup-grafana/configure-security/configure-authentication/" >}}), in the instructions for your Oauth identity provider.
```

File: docs/sources/whatsnew/whats-new-in-v9-3.md [hunk 4]
```diff
@@ -73,3 +73,13 @@ With the Partition by values transformer, you can now issue a single query and s
 | ------------------- | ------ | ----- |
 | 2022-10-20 12:00:00 | EU     | 2936  |
 | 2022-10-20 01:00:00 | EU     | 912   |
+
+## Authentication - OAuth token handling improvements
+
+This feature is generally available in Grafana Open Source, Enterprise, Cloud Free, Cloud Pro, and Cloud Advanced.
+
+As part of our efforts to improve security of Grafana, we introduce a long-awaited feature which enhances Grafana's OAuth 2.0 compatibility. When a user logs in using an OAuth provider, Grafana verifies on each request that the access token has not expired. When an access token expires, Grafana uses the provided refresh token (if any exists) to obtain a new access token.
```
Comment (sarahleejane) [inline docs/sources/whatsnew/whats-new-in-v9-3.md:81]: As part of our efforts to improve the security of Grafana, we are introducing a long-awaited feature which enhances Grafana's OAuth 2.0 compatibility. When a user logs in using an OAuth provider, on each request Grafana verifies that the access token has not expired. When an access token expires, Grafana uses the refresh token provided (if any exists) to obtain a new access token.

**Pull request merged**